### ISSUE TITLE
Duplicate IDs, Aria-labels and Labels in Example and test files.

### ISSUE THREAD

**Issue by saket2403** [cerner/terra-core#2709]
## Description
Examples and Test Files of certain components have duplicate IDs, Aria-labels and label , thus giving violations while running the axe plugin.

### Components
- Form   Input
- Form Select
- Status View
- Toggle
- Form Textarea
